dependency/dependency_test.go

```diff
@@ -58,18 +51,20 @@ func TestMain(m *testing.M) {
 		},
 	}
 
-	if err := testClients.consul.client.Agent().ServiceRegister(serviceMetaService); err != nil {
-		panic(err)
+	consul_agent := testClients.consul.client.Agent()
```

Comment:
Snake case names should be replaced with camel case

Reply:
I don't agree with using camelCase for everything and snake_case for nothing. I think snake_case for local variables is perfectly fine.